arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'How to Deploy a Token Bridge Using the Orbit SDK'
+title: 'Deploy a token bridge using the Orbit SDK'
```

Comment:
sentence-casing ([ref](https://docs.arbitrum.io/for-devs/contribute#style-conventions))